tests/Fixer/NamespaceNotation/NoBlankLinesBeforeNamespaceFixerTest.php

```diff
@@ -44,7 +49,8 @@ public function provideFixCases()
             array("<?php\nnamespace X;"),
             array("<?php\nnamespace X;", "<?php\n\n\n\nnamespace X;"),
             array("<?php\r\nnamespace X;"),
-            array("<?php\r\nnamespace X;", "<?php\r\n\r\n\r\n\r\nnamespace X;"),
+            array("<?php\nnamespace X;", "<?php\r\n\r\n\r\n\r\nnamespace X;"),
+            array("<?php\r\nnamespace X;", "<?php\r\n\r\n\r\n\r\nnamespace X;", new WhitespacesFixerConfig('    ', "\r\n")),
```

Comment:
Maybe it would be nice to also add the test case with namespace right after the opening tag here, i.e.:
```php
array('<?php namespace X;'),
```

Reply:
Do you mean this?

```php
array('<?php\nnamespace X;', '<?php namespace X;'),
```

Reply:
If `NoBlankLinesBeforeNamespaceFixer` should keep `<?php namespace X;` untouched, the `$expected` parameter in 
`fixLinesBeforeNamespace` isn't enough: we need a way to differentiate *I want exactly 1 new line before namespace* and *I want up to 1 new line before namespace*

Reply:
Ah sorry, I thought the fixer would not change anything in such case. Then you can ignore my previous comment.